package.json

```diff
@@ -49,7 +49,7 @@
     "release": "node scripts/release.js",
     "trace": "node -r esm bin-src/trace.js",
     "trim-stats": "node -r esm bin-src/trim-stats-file.js",
-    "storybook": "start-storybook -p 9009 -s static",
```

Comment:
Don't remove `-s`, Chromatic doesn't support `staticDirs` in main.js yet (for TurboSnap). Sucks, I know.